src/Serilog/Serilog.csproj

```diff
@@ -1,9 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Simple .NET logging with fully-structured events</Description>
     <VersionPrefix>2.12.1</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.0;net45;net46;net47;net5.0;net6.0</TargetFrameworks>
```

Comment:
May not be much point yet adding this, since our dependencies and features will match net6.0 exactly, and it's the LTS.

Reply:
Should we consider removing `net5.0` since it's out of support

Reply:
It gives a very little benefit. Dropping `netstandard1.0;net45;etc` will give more noticeable benefits. I would not drop so.